rclcpp/include/rclcpp/node.hpp

```diff
@@ -1072,6 +1072,58 @@ class Node : public std::enable_shared_from_this<Node>
   bool
   assert_liveliness() const;
 
+  /// Return a list of publishers about topic endpoint information to a given topic.
```

Comment:
nitpick: This is a very awkward sentence.

Maybe "Return the topic endpoint information about publishers on a given topic."?

Reply:
Done.